app/lib/aid_application_cleaner.rb

```diff
@@ -0,0 +1,14 @@
+class AidApplicationCleaner
+  def clean_production
+    if Rails.env.production?
+      apps_to_delete = AidApplication.where(submitted_at: nil).where('created_at < ?', 1.day.ago)
+      apps_to_delete.destroy_all
+    end
+  end
```

Comment:
I would like this method to be much more declarative about what it's doing. I also think you can remove the production flag because it should be relatively "safe" to run anywhere/anytime.

Something like "delete_stale_and_unsubmitted"

Reply:
did you want the method to be renamed or the variable inside the method?

Reply:
The method to be renamed.